src/main/java/com/google/sps/data/ArticleBuilder.java

```diff
@@ -0,0 +1,19 @@
+package com.google.sps.data;
+
+public class ArticleBuilder {
+    public String title;
+    public String link;
+
+    public ArticleBuilder(String title) {
+      this.title = title;
+    }
+
+    public ArticleBuilder withLink(String link) {
+      this.link = link;
+      return this;
+    }
```

Comment:
It's generally also good practice to have your instance fields private (unless they're constants, in which case it mostly depends on what you're trying to do), especially since they're not _final._ 

You can also consider having an empty constructor, with another method returning ArticleBuilder that sets the title instead.